daemon/server/scootAPI.go

```diff
@@ -0,0 +1,16 @@
+package server
```

Comment:
This file doesn't appear to be necessary. We want the interface to be defined in daemon/protocol/daemon.proto (or thrift if we choose to switch), and to implement it using already existing components like runner.Runner (which will happen to be a RunnerManager in the daemon).